mac/Makefile.am

```diff
@@ -4,6 +4,6 @@ libhidapi_la_LDFLAGS = $(LTLDFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir)/hidapi/
 
 hdrdir = $(includedir)/hidapi
-hdr_HEADERS = $(top_srcdir)/hidapi/hidapi.h
+hdr_HEADERS = $(top_srcdir)/hidapi/hidapi.h hidapi_darwin.h
```

Comment:
revert

autotools are deprecated now
we don't want to develope/support new feature for it - that will be a good motivation to migrate to CMake for projects that are still using it autotools and are actively developed